test/test_other.py

```diff
@@ -11483,7 +11482,7 @@ def test_split_main_module(self):
 
     self.run_process([EMCC, side_src, '-sSIDE_MODULE', '-g', '-o', 'libhello.wasm'])
 
-    self.emcc_args += ['-g']
+    self.emcc_args += ['-g', 'libhello.wasm']
```

Comment:
Why is there an extra input file here? I guess to scan the symbols for error reporting?

Reply:
Actually this wasn't needed in the end. Reverted.